src/public/js/util.js

```diff
@@ -26,4 +26,24 @@ const downloadBlob = function (data, fileName, mimeType) {
   }, 1000);
 };
 
-export { logout, downloadBlob };
+/**
+ * 
+ * @param {string} selector 
+ */
+const hideElement = function (selector) {
+  document.querySelectorAll(selector).forEach(el => {
+    el.style.display = 'none';
+  });
+}
+
+/**
+ * 
+ * @param {string} selector 
+ */
+const showElement = function (selector) {
+  document.querySelectorAll(selector).forEach(el => {
+    el.style.display = 'block';
```

Comment:
This is not ideal as not all elements may be block-level, instead set the display style property to an empty string.